src/lib/common/limits.h

```diff
@@ -94,6 +95,7 @@
 */
 #define  LOCAL_IP_V6  "::"
 #define  LOCAL_IP_V4  "0.0.0.0"
+#define  MAX_LEN_IP   64
```

Comment:
The reational behind this limit calculation should be added as comment around this line... It also works for IPv6 addresses (it seems so, but think about it for a moment...)?